Xamarin.Forms.Controls/ViewContainers/StateViewContainer.cs

```diff
@@ -24,8 +24,11 @@ public StateViewContainer (Enum formsMember, T view) : base (formsMember, view)
 				BindingContext = view,
 				AutomationId = name + "StateLabel"
 			};
-			if (name != "Focus")
-				stateValueLabel.SetBinding (Label.TextProperty, name, converter: new GenericValueConverter (o => o.ToString()));
+
+			if(name == "Focus" || name == "Unfocused" || name == "Focus")
```

Comment:
Did you mean to put `name == "Focus"` twice?

```suggestion
			if (name == "Focus" || name == "Unfocused")
```

Reply:
- Focused